packages/material-ui-utils/src/deepmerge.js

```diff
@@ -2,6 +2,10 @@ export function isObject(item) {
   return item && typeof item === 'object' && !Array.isArray(item);
 }
 
+function isElement(element) {
+  return element instanceof Element || element instanceof HTMLDocument;
```

Comment:
Will it work on server side?

Reply:
Damn, you're right. 😞

I'll add a check to ensure this is only run in the browser.

Reply:
What about
```js
function isElement(element) {
  if(typeof window === "undefined") {
    return false
  }
  return element instanceof Element || element instanceof HTMLDocument;  
}
```
?